libs/blocks/graybox/graybox.js

```diff
@@ -264,7 +264,16 @@ const setupChangedEls = (globalNoClick) => {
   }
 };
 
+const transformLinks = () => {
+  const grayboxHostname = window.location.hostname;
```

Comment:
There isn't a need for local development where ports would also be involved, right?

Reply:
Correct, no ports are involved for this. So, I think we’re good with `.hostname`